AK/Statistics.h

```diff
@@ -29,6 +29,28 @@ requires(IsArithmetic<T>) class Statistics {
     T const sum() const { return m_sum; }
     float average() const { return (float)sum() / size(); }
 
+    T const min()
+    {
+        T min_v = m_values[0];
```

Comment:
What if it's empty?

Reply:
Correct me if I'm wrong, isn't the same can be said for the rest of the methods? (`average()`, `standard_deviation()`, `median()` etc.) There should be a common behaviour, I can implement and commit it when decided.

Reply:
Oh they all do this? Sorry, I didn't check the other ones.
Regardless, I think it makes more sense to not store the values at all, unless we want some of the more...specialised statistical measures.

Reply:
I believe we already need it for `.median()`, right? I haven't heard of an online algorithm to calculate median.

Reply:
:thinking: even for that, we can use a richer data structure like `HashMap<T, size_t>`, mapping a value to how many times it's been encountered.

Reply:
Also related, storing values is _necessary_ if we want to implement a `.delete()` method in the future. HashMap sounds much much better for it anyway.

Reply:
It might be better here to use something like a red-black tree instead of a HashMap here. Insertions would be a bit slower (I don't suspect dealing with truly large amounts of data... for now at least), but you would be able to efficiently loop over the data in sorted order to be able to get medians / k-th order statistics as well.